modules/calcite/src/test/sql/types/decimal/cast_from_decimal.test_ignored

```diff
@@ -0,0 +1,78 @@
+# name: test/sql/types/decimal/cast_from_decimal.test
+# description: Test casting from decimal to numeric
+# group: [decimal]
```

Comment:
What about link to jira issue for ignored tests (here and below)?
I guess we have to add at least one link at the header. If need some queries may be commented with separate issues too.

Reply:
Fixed